autoscheduler/frontend/src/components/SchedulingPage/CourseSelectColumn/CourseSelectCard/CourseSelectCard.tsx

```diff
@@ -4,18 +4,27 @@ import RemoveIcon from '@material-ui/icons/Delete';
 import CollapseIcon from '@material-ui/icons/ExpandLess';
 import Autocomplete from '@material-ui/lab/Autocomplete';
 import {
-  TextField, ButtonGroup, Button, FormLabel, Card, Typography, Collapse, Switch,
+  TextField, Card, Collapse as CollapseBase, Switch,
 } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
 import { RootState } from '../../../../redux/reducer';
 import { updateCourseCard } from '../../../../redux/actions/courseCards';
-import { CustomizationLevel, CourseCardOptions } from '../../../../types/CourseCardOptions';
+import { CourseCardOptions } from '../../../../types/CourseCardOptions';
 import SmallFastProgress from '../../../SmallFastProgress';
 
 import * as styles from './ExpandedCourseCard/ExpandedCourseCard.css';
 import SectionSelect from './ExpandedCourseCard/SectionSelect/SectionSelect';
-import BasicSelect from './ExpandedCourseCard/BasicSelect/BasicSelect';
 import { getCourseCardHeaderColor } from '../../../../theme';
 
+const Collapse = withStyles({
+  container: {
+    display: 'flex',
+  },
+  wrapper: {
+    width: '100%',
+  },
+})(CollapseBase);
+
```

Comment:
I noticed that this PR breaks the transition for course cards (cards opening "snap" open all the way open from closed at the very end of the animation). This branch is on the left, production is on the right:
![animationbroken](https://user-images.githubusercontent.com/28655462/121786759-9f62ae00-cb87-11eb-8a61-2d856d4f1ad8.gif).

Removing this override for the `Collapse` component seemed to fix this issue for me, why is it here considering it just seems to break the animation?

Reply:
When its removed it breaks scrolling, let me test some alternate solutions.

Reply:
This is fixed now